### DISCUSSION TITLE
interface is displayed down

### DISCUSSION THREAD

**Discussion by buzz3rro** [bsmithio/OPNsense-Dashboard#10] [Q&A]
first of all thanks for this really awesome dashboard. I installed and set it up according to your instructions. 
It looks really nice.

I have only one small issue, the interfaces in the lower dashboard are displayed as "down". On the top everything is fine.
I do not understand what I have entered here wrong. Does anyone have an idea what is wrong here?

upper part: displayed interface "up"
![image](https://user-images.githubusercontent.com/97843718/153139455-8b4b68ad-b893-4858-b98b-b1eecbdb4783.png)

lower part: displayed interface "down" also the "friendly name is not up to date.
![image](https://user-images.githubusercontent.com/97843718/153139819-44189b50-e65d-4fda-9622-2825939e727e.png)

**Comment by bsmithio** [comment]
Hi, I believe this was due to a wrong query I had in the interface panels. The query wasn't pulling the latest status. I've updated the dashboard to fix these queries. You will need to re-import the dashboard for the new changes.
